<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="moduleBar">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="moduleHeader">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="modulesArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="modulesAreaContent">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>278</width>
           <height>622</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="addTap">
            <property name="text">
             <string>Add &quot;tap&quot; Module</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addPress">
            <property name="text">
             <string>Add &quot;press&quot; Module</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addRelease">
            <property name="text">
             <string>Add &quot;release&quot; Module</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addWait">
            <property name="text">
             <string>Add &quot;wait&quot; Module</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addMoveAbsolute">
            <property name="text">
             <string>Add &quot;move-absolute&quot; Module</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addMoveRelative">
            <property name="text">
             <string>Add &quot;move-relative&quot; Module</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addExec">
            <property name="text">
             <string>Add &quot;exec&quot; Module</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addLoop">
            <property name="text">
             <string>Add &quot;loop&quot; Module</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addIf">
            <property name="text">
             <string>Add &quot;if&quot; Module</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addModifyVariable">
            <property name="text">
             <string>Add &quot;modify-variable&quot; Module</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scriptArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <widget class="QWidget" name="scriptAreaContent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>392</width>
        <height>748</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="configBar">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
